### PR TITLE
Add Bastien on Lightning Privacy

### PR DESCRIPTION
**Pull request by freddiekrugerrand**

### CODE REVIEW

**Review by adamjonas — COMMENTED** [review]
Thanks for this @freddiekrugerrand. A few fix-ups but looking forward to getting this in.

File: adopting-bitcoin/2021/2021-11-17-bastien-teinturier-privacy-on-lightning.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+transcript_by: freddiekrugerrand
+categories: ['conference']
+tags: ['lightning', 'privacy']
+---
+
+Name: Bastien Teinturier
+
+Topic: Privacy On Lightning
+
+Video: https://www.youtube.com/watch?v=gOiKlx9a1xg
```
[the remark this answers is not in the document]
Comment (freddiekrugerrand) [reply adopting-bitcoin/2021/2021-11-17-bastien-teinturier-privacy-on-lightning.md]: Done!

File: adopting-bitcoin/2021/2021-11-17-bastien-teinturier-privacy-on-lightning.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
...adcast this red transaction on-chain, and since it's using pay to witness script hash, people watching the chain do no know it's a Lightning channel. It just looks like any other pay to witness script hash on the network. When you close the channel and spend that output, you have to reveal the script that is wrapped in the pay to witness script hash. At that point everybody watching the chain knows that this was a 2-of-2 multi-sig, and these are not very common at all. When people see that on-chain, they can be pretty sure that this was a Lighting channel. This is bad. This is the kind of thing we want to avoid. Luckily, this one is very easy to fix with Taproot. I'm saying Taproot in many places, but we're actually using Schnorr, not the script part of Taproot. It's just here we will change the transaction, instead of using a pay to witness script hash we will use what Jonas just talked about, musig2. Instead of exposing the two different keys that went into this multi-signature, we'll be using musig so it will just look like any other key path single signature spend on the network. Anyone looking at the chain will just see that this is one Schnorr signature, it could be anything.
```
[the remark this answers is not in the document]
Comment (freddiekrugerrand) [reply adopting-bitcoin/2021/2021-11-17-bastien-teinturier-privacy-on-lightning.md]: Is there a easy way to see what the suggested change is in these? Difficult to detect the typo in the big block github displays.

**Comment by freddiekrugerrand** [comment]
Thanks for the review @adamjonas! I think I've squashed all the typos, difficult to see where the problem is in some of the suggestion blocks though.

File: adopting-bitcoin/2021/2021-11-17-bastien-teinturier-privacy-on-lightning.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kouloumos) [reply adopting-bitcoin/2021/2021-11-17-bastien-teinturier-privacy-on-lightning.md]: Try https://text-compare.com/

**Pull request merged**